pkgs/tools/admin/google-cloud-sdk/update.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+BASE_URL="https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk"
+
+# Version of Google Cloud SDK from
+# https://cloud.google.com/sdk/docs/release-notes
+VERSION="364.0.0"
+
+function genMainSrc() {
+    local url="${BASE_URL}-${VERSION}-${1}-${2}.tar.gz"
+    local sha256
+    sha256=$(nix-prefetch-url "$url")
+    echo "      {"
+    echo "        url = \"${url}\";"
+    echo "        sha256 = \"${sha256}\";"
+    echo "      };"
```

Comment:
That should be added to the nix-shell.

Reply:
It would be really helpful if you can point to any existing script that works that way.

Reply:
Probably any update.sh uses nix-shell https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/ombi/update.sh

Reply:
I added `nix` to the shell package, cause `nix-prefetch-url` is the only one that's used.